Update fantical support for AWS references (#133)
rackerlabs/docs-aws: docs/user-management-and-perms/index.rst

^^^^^^^^^^^^^^^^^^^^^^

These permissions are Rackspace product-specific. This is where you manage

> Product Access. This guide doe not cover other product permissions.

Reviewer: does

Author: ```suggestion
Product Access. This guide does not cover other product permissions.
```

There are three Fanatical Support for AWS permissions:

* **Allow this user to add AWS Accounts**: Enables the user to add
  additional AWS accounts for Rackspace to manage. These could be new or
  existing AWS accounts.

> * **Product Access**: Controls what access the user has within the

Reviewer: I recently found out that we do not need the word Technology in Rackspace Technology Customer Portal.

>   `Rackspace Technology Customer Portal <https://manage.rackspace.com/aws>`_.

Reviewer: Turn out we don't need the word "Technology" in Rackspace Customer Portal.

>   `Rackspace Technology Customer Portal <https://manage.rackspace.com/aws>`_.

Author: ```suggestion
  `Rackspace Customer Portal <https://manage.rackspace.com/aws>`_.
```

  credentials. You can manage this permission for any AWS managed or custom IAM
  policy available on the AWS account and configure it for each AWS account.

Accounts with Identity Federation
---------------------------------

> Users who are provisioned manually via the Rackspace portal (none-federated users) can not inherit **Product Access**

Reviewer: non-federated

Reviewer: can not should cannot

Author: ```suggestion
Users who are provisioned manually via the Rackspace portal (non-federated users) cannot inherit **Product Access**
```

**App1-Staging**

* ``Admin`` access to **Product Access** so, for example, he can
  authenticate to instances through Passport.

> * ``AdministratorAccess`` IAM policy access so he has full access to AWS

Author: ```suggestion
* ``AdministratorAccess`` IAM policy access so the user has full access to AWS
```

  services through the **AWS Console and APIs**.

> 

Reviewer: change 'he' to 'the user'
